contracts/StablecoinConverter.sol

```diff
@@ -122,6 +122,7 @@ contract StablecoinConverter is EpochTokenLocker {
         uint batchId;
         TradeData[] trades;
         uint16[] tokenIdsForPrice;
+        address solutionSubmitter;
```

Comment:
Instead of having currentFeeCollected separate, can we store both rewardAmount and rewardReceiver on this struct?